flink-connectors/flink-hbase/pom.xml

```diff
@@ -340,6 +340,25 @@ under the License.
 				</dependencies>
 			</dependencyManagement>
 		</profile>
+		<profile>
+			<id>java11</id>
+			<activation>
+				<jdk>11</jdk>
+			</activation>
+
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<!-- hive does not support Java 11 -->
```

Comment:
Minor: hive->hbase, and suggest to mark as "currently doesn't support Java 11" due to HBASE-21110 is ongoing.